requirements/edx/coverage.txt

```diff
@@ -4,7 +4,8 @@
 #
 #    make upgrade
 #
-coverage==4.4
+git+https://github.com/nedbat/coveragepy.git@e9e703c7885840412295a0aaf3c84b5244483881#egg=coverage==5.0a6
```

Comment:
I was looking at branches in coverage.py I could delete, and came here to see what coverage is pinned to.  This hash seems too old (it's before the numbits line number storage).